docs/estela-cli/index.md

```diff
@@ -2,7 +2,15 @@
 layout: page
 title: estela CLI
 nav_order: 4
-has_children: true
+has_children: false
 ---
 
 # estela CLI
+
+The **estela CLI** is a command line client designed to interact with the estela API. It enables users to perform various actions:
```

Comment:
Let's change `command line client` to `command line interface`, and we can merge this.